lib/src/google_fonts_base.dart

```diff
@@ -99,7 +107,9 @@ TextStyle googleFontsTextStyle({
     file: fonts[matchedVariant]!,
   );
 
-  loadFontIfNecessary(descriptor);
+  final loadingFuture = loadFontIfNecessary(descriptor);
```

Comment:
test cases would be nice to validate what this behaves like when loading the font isn't necessary, for instance

Reply:
I think this has been covered by `load_font_if_necessary_with_local_fonts_test.dart`.